src/content/docs/releases/changelog.mdx

```diff
@@ -586,6 +586,15 @@ import Aside from '@components/Aside.astro';
 **Payment Services** 
 ********************/}
 
+    <ChangelogEntry
+      date="2025-10-29"
+      title="Payment Services v1.0.3"
+      components={['Payment Services']}
+    >
+      - **Fixed a server error during Credit Card checkout on storefronts connected to Adobe Commerce Cloud Service (ACCS).**
```

Comment:
I'd reframe to something as "Removed a redundant call to "setPaymentMethodOnCart" to speed up the checkout and make it resilient to server-side errors"

Technically, there are no server-side errors in the droppin itself 🤷🏻

Reply:
What about
> Removed a redundant call to `setPaymentMethodOnCart` mutation that was failing for Credit Card checkouts in storefronts connected to Adobe Commerce Cloud Service (ACCS) due to issue [CCSAAS-3786](https://jira.corp.adobe.com/browse/CCSAAS-3786).